docs/ppl/index.md

```diff
@@ -9,6 +9,7 @@ has_toc: false
 # PPL
 
 Piped Processing Language (PPL) is a query language that makes it easier to query data stored in Elasticsearch as compared to the standard domain-specific language (DSL).
+PPL lets you use pipe (I) syntax to explore, discover, and query data stored in Elasticsearch.
```

Comment:
I don't think that's quite the character we want. Isn't it (\|)?

Reply:
Yep, missed this. Fixed.